tpl/config.js

```diff
@@ -117,6 +132,33 @@ const config = {
     `),
     encoderName: 'TransferTxData',
   },
+  //[fieldName, filedBits, releaxed (number not need to be fitted in field Bits)]:
+  dataEncodeSchemes: {
+    common: [
+      ['accountID1', 'accountLevels'],
+      ['accountID2', 'accountLevels'],
+      ['tokenID1', 'balanceLevels'],
+      ['tokenID2', 'balanceLevels'],
+      ['amount', 'floats'],
+    ],
+    spotTrade: [
+      ['accountID1', 'accountLevels'],
+      ['accountID2', 'accountLevels'],
+      ['tokenID1', 'balanceLevels'],
+      ['tokenID2', 'balanceLevels'],
+      ['newOrder1AmountSell', 'floats'],
+      ['newOrder1AmountBuy', 'floats'],
+      ['newOrder1ID', 'orderLevels', true],
+      ['newOrder2AmountSell', 'floats'],
+      ['newOrder2AmountBuy', 'floats'],
+      ['newOrder2ID', 'orderLevels', true],
+    ],
+    l2Key: [
+      ['accountID1', 'accountLevels'],
+      ['sign2', '1'],
+      ['ay2', '254'],
```

Comment:
TBH it feels so weird that we sometimes use `1` (`accountID1`), sometimes use `2` (`sign2`, `ay2`).

Reply:
I get your points now. LGTM.